custom_components/tuya_local/devices/unknow_2way_energymeter.yaml

```diff
@@ -0,0 +1,452 @@
+# Can´t put some DPs to work (calibration and alarms)
+name: Bidirectional energy meter
+products:
+  - id: 8bjnabt4gh8tu4vr
+    name: Smart meters
+primary_entity:
+  entity: sensor
+  name: Forward energy total
+  class: energy
+  dps:
+    - id: 1
+      type: integer
+      optional: true
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      mapping:
+        - scale: 100
+secondary_entities:
+  - entity: sensor
+    name: Forward energy total hour
+    class: energy
+    dps:
+      - id: 122
+        type: integer
+        optional: true
+        name: sensor
+        unit: kWh
+        class: total_increasing
+        mapping:
+          - scale: 100
+  - entity: sensor
+    name: Reverse energy total
+    class: energy
+    dps:
+      - id: 2
+        type: integer
+        optional: true
+        name: sensor
+        unit: kWh
+        class: total_increasing
+        mapping:
+          - scale: 100
+  - entity: sensor
+    name: Reverse energy total hour
+    class: energy
+    dps:
+      - id: 123
+        type: integer
+        optional: true
+        name: sensor
+        unit: kWh
+        class: total_increasing
+        mapping:
+          - scale: 100
+  - entity: sensor
+    class: voltage
+    category: diagnostic
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: V
+        class: measurement
+        mapping:
+          - scale: 10
+      - id: 127
+        type: integer
+        name: calibration
+        step: 1
```

Comment:
step is only valid inside a mapping, 1 is the default step, and this attribute cannot be modified in HA, so step is irrelevant, so remove this line (and any others that match one or more of the above reasons)